xcell/tests/test_mapper_dummy.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import xcell as xc
+import healpy as hp
+
+
+def get_config(spin=0):
+    return {'l0': 100,
+            'nside': 32,
+            'alpha': -2,
+            'spin': spin}
+
+
+def get_mapper():
+    return xc.mappers.MapperDummy(get_config())
+
+def test_smoke():
+    get_mapper()
+
+
+def test_get_mask():
+    m = get_mapper()
+    ns = m.nside
+    d = m.get_mask(ns, fsk=1)
+    assert np.all(d == 1)
+    d = m.get_mask(ns, fsk=100)
+    assert np.all(d == 1)
+    # TODO: Implement apodized mask
+    # d = m.get_mask(fsky=0.5)
+
+def test_get_cl():
+    m = get_mapper()
+    c = get_config()
+    ls = 120
+    cl = 1./(ls+c['l0'])**c['alpha']
+    assert(np.fabs(cl - m.get_cl(ls))< 1E-5)
+
+def test_get_signal_map():
+    m = get_mapper()
+    m.seed = 0
+    d = m.get_signal_map()
+    d0 = [np.loadtxt('xcell/tests/data/dummy_signal_s0.txt')]
```

Comment:
is `d0` a map? If so, it'd be better if it's saved in a fits file (it will preserve floating point precision and probably be smaller)

Reply:
Also, I'm not sure this will always work (i.e. not sure if setting the same seed in different machines leads to the same map), but we'll see!